examples/marketing-model/ml_marketing_banking_model.ipynb

```diff
@@ -0,0 +1,1902 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Stefan - How to build a Marketing Model for Banking using Google Cloud Version 0.1.ipynb",
```

Comment:
This name probably shouldn't contain your name and the title can be made shorter like the above.